### PR TITLE
security guidance

### LINKED ISSUE #20: How do we expect organizers to manage passwords?
We tell organizers to e.g. set up Twitter/Facebook using the chapter email address.

Now, how do we expect them to share _access_ to these accounts? Do they simply share a password and try to renew it often? If a single person manages the Twitter account with 2FA, where are the recovery codes for 2FA stored in case this person unexpectedly disappears?

For Twitter, with Tweetdeck you can add users to an account without their knowing the password.

### PR DESCRIPTION
**Pull request by maelle**
Fix #20 

@Athanasiamo would you mind reviewing and adding tips about the password manager you use?

### CODE REVIEW

**Comment by maelle** [comment]
note to self: add https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008563 recommended by @yabellini

File: content/organization/tech/security/index.en.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Security"
+menuTitle: "Security"
+weight: 54
+---
+
+As organizers you will be responsible for accounts such as email, social media accounts.
```
Comment (Athanasiamo) [inline content/organization/tech/security/index.en.md:7]: email and social media accounts

File: content/organization/tech/security/index.en.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
```
Comment (Athanasiamo) [inline content/organization/tech/security/index.en.md:14]: For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or [Bitwarden](https://bitwarden.com/), or a paid service like [1Password](https://1password.com/).

File: content/organization/tech/security/index.en.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+If you feel on the fence, ask other organizers or your friends about their experience with some password managers.
+
+### KeePass workflow
+
+* save all your passwords in a KeePass database.
+* have it backed-up on Dropbox or Google Drive.
+* only learn two passwords: the one to decrypt the KeePass database, the one to your Google Drive account.
+
```
Comment (Athanasiamo) [inline content/organization/tech/security/index.en.md:23]: ### Bitwarden workflow

Free for single person, or two person 'organisation' (ie. ability share certain passwords with a single other person). 
Download the Desktop client and/or the extension/plugin for your browser of choice. 
Choose a password you can remember for Bitwarden, and store all your passwords in there. 
Also has biometrics integration for those using it.

File: content/organization/tech/security/index.en.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+As organizers you will be responsible for accounts such as email, social media accounts.
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
+
+If you feel on the fence, ask other organizers or your friends about their experience with some password managers.
+
```
Comment (Athanasiamo) [inline content/organization/tech/security/index.en.md:17]: ### Common password manager workflow

Password managers make it safe to store all your passwords in encrypted, safe storage. 
You only need to remember the password to unlock your password manager, to get access to all your account passwords.
This means this "master password" needs to be safe, meaning:
- Be at least several words long (using phrases from shows/songs you like ight be a good way to go)
- Contain special characters and numbers
- Be hard to guess, but easy for you to remember
- Not used in _any other instance_.

File: content/organization/tech/security/index.en.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Make sure to save the recovery codes of 2FA in your personal password manager, and/or to share them with your team.
+
+Note that apps like DuoMobile allow you to sync your account with GoogleDrive which can be very handy when changing phones for instance.
+
+## Share passwords and 2FA recovery codes securely with co-organizers
+
+Your chapter will have a bunch of passwords and 2FA recovery codes to share.
+How to do that?
+
+* You might try and apply for 1Password OSS free plan.
```
Comment (Athanasiamo) [inline content/organization/tech/security/index.en.md:42]: * You might try and apply for [1Password OSS free plan](https://github.com/1Password/1password-teams-open-source)

**Review by Athanasiamo — APPROVED** [review]

**Comment by maelle** [comment]
Thanks a ton @Athanasiamo!

**Pull request merged**